src/algokit/cli/init.py

```diff
@@ -216,6 +217,12 @@ def init_command(  # noqa: PLR0913
     # provide the directory name as an answer to the template, if not explicitly overridden by user
     answers_dict.setdefault("project_name", directory_name)
 
+    system_python_path = next(get_python_paths(), None)
```

Comment:
Should we do the same for the generators? It's possible a generator might try to execute a python script via copier.

Reply:
@negar-abbasi Unmarked this as resolved, as it doesn't appear like this has been handled.

Reply:
I thought we are handling it with docs, telling user if they need this option then they have to do use 'python_path' convention as we did.

Reply:
But we won't be resolving the `python_path` inside the `generate` command, so it won't work inside a generator.

Reply:
@negar-abbasi @neilcampbell see my last commit that adds the same logic to generate (i am not covering other default answers yet only the python_path answer for now). And there is an extra snapshot test that runs generate without mocking and tests that there is non zero exit code.